src/electionguard-ui/ElectionGuard.UI/Views/TallyProcessPage.xaml

```diff
@@ -233,6 +240,11 @@
                     Text="{helper:Translate TallySubtalliesCombined}"
                     VerticalTextAlignment="Center"
                     WidthRequest="200" />
+                <ActivityIndicator
+                    Margin="10"
+                    HeightRequest="100"
+                    IsRunning="{Binding Checklist.TallyStarted}"

```

Comment:
Do you need this binding also on isvisible?

Reply:
unsure so let's be explicit